pyant/plotting.py

```diff
@@ -233,18 +233,22 @@ def gain_heatmap(beam, polarization=None, resolution=201, min_elevation=0.0, lev
     SdB[np.isinf(SdB)] = 0
     SdB[np.isnan(SdB)] = 0
     SdB[SdB < 0] = 0
-    conf = ax.contourf(K[:,:,0], K[:,:,1], SdB, cmap=cm.plasma, vmin=0, vmax=np.max(SdB), levels=levels)
+    # conf = ax.contourf(K[:,:,0], K[:,:,1], SdB, cmap=cm.plasma, vmin=0, vmax=np.max(SdB), levels=levels)
+    conf = ax.pcolormesh(K[:,:,0], K[:,:,1], SdB, cmap=cm.plasma, vmin=0, vmax=np.max(SdB), shading='giraud')
+
     if usetex:
-        ax.set_xlabel('$k_x$ [1]',fontsize=24)
-        ax.set_ylabel('$k_y$ [1]',fontsize=24)
+        ax.set_xlabel('$k_x$ [1]') #,fontsize=24)
+        ax.set_ylabel('$k_y$ [1]') #,fontsize=24)
     else:
-        ax.set_xlabel('kx [1]',fontsize=24)
-        ax.set_ylabel('ky [1]',fontsize=24)
```

Comment:
fontsize should be a input option maybe?

Reply:
the labels were large enough that the plots were unviewable on my screen, is all.  If you need to change such things on a regular basis, I'd suggest you look at rcparam settings for matplotlib

Reply:
Noted: i have usually high resolution on my main screen.

Resolution: creating issue to add a custom default style and allow editing of this style live, the style can be applied with `plt.rcParams.update(pyant_style)` like command